CONTRIBUTING.md

```diff
@@ -73,6 +73,19 @@ Packetbeat:
 Some of the Beats might have extra development requirements, in which case a
 CONTRIBUTING.md file is find in the Beat directory.
 
+## Update scripts
+
+The beats use a variety of scripts based on python to generate configuration files
+and documentations. The command used for this is:
+
+    $ make update
+
+This command has the following dependencies:
+
+* Python >=2.7.9
+* [virtualenv](https://virtualenv.pypa.io/en/latest/) for Python
```

Comment:
We could add here the command to get it (`easy_install virtualenv` or `pip install virtualenv`) for people unfamiliar with how the python stuff works.